src/models/Coordinate.tsx

```diff
@@ -0,0 +1,14 @@
+export class Coordinate {
```

Comment:
Do we need this class?

At least, for now, this could be just a type. What do you think?
```
type Coordinate = {
    Lon: number;
    Lat: number;
}
```

Reply:
I was thinking that maybe we could just use { lon: number, lat: number } where we are pointing to this class.